templates/app/controllers/orders_controller.rb

```diff
@@ -50,9 +50,11 @@ def empty
       @order.empty!
     end
 
-    redirect_to cart_path
+    redirect_to edit_cart_path
   end
 
+  private
```

Comment:
I honestly didn't want to create a new PR just to make `accurate_title` private :p